twoliter/embedded/rpm2img

```diff
@@ -240,6 +240,37 @@ INVENTORY_DATA="$(jq --arg PKG_PREFIX "bottlerocket-" \
 INVENTORY_DATA="$(jq --slurp 'sort_by(.Name)' <<< "${INVENTORY_DATA}" | jq '{"Content": .}')"
 printf "%s\n" "${INVENTORY_DATA}" > "${ROOT_MOUNT}/usr/share/bottlerocket/application-inventory.json"
 
+elf_note() {
+  local input output size offset uint32_t
+  input="${1:?}"
+  output="${2:?}"
+  name="Bottlerocket"
+  name_len="$(( ${#name} + 1 ))"
+  input_len="$(( ${#input} + 1 ))"
+  rm -f "${output}"
+  for n in "${name_len}" "${input_len}" 0 ; do
+    printf -v uint32_t '\\x%02x\\x%02x\\x%02x\\x%02x' \
+      $((n & 255)) $((n >> 8 & 255)) $((n >> 16 & 255)) $((n >> 24 & 255))
+    printf "${uint32_t}" >> "${output}"
+  done
+  echo -en "${name}\x00" >> "${output}"
+  echo -en "${input}\x00" >> "${output}"
+}
+
+# add elf note to relevant binaries
+elf_note "${VARIANT}" "note.elf"
+
+ELF_MAGIC="\x7fELF"
+for file in "${ROOT_MOUNT}"/usr/bin/*; do
```

Comment:
```suggestion
for some_file in "${ROOT_MOUNT}"/usr/bin/*; do
```
Can we avoid `file` as a variable name? There is `file` utility present in the Bottlerocket SDK.